free-programming-books-ja.md

```diff
@@ -477,15 +476,10 @@
 
 ### OCaml
 
-* [Objective Caml 入門](http://www.fos.kuis.kyoto-u.ac.jp/~t-sekiym/classes/isle4/mltext/ocaml.html) - 五十嵐淳
+* [Objective Caml 入門](https://web.archive.org/web/20161128072705/http://www.fos.kuis.kyoto-u.ac.jp:80/~t-sekiym/classes/isle4/mltext/ocaml.html) - 五十嵐淳
 * [お気楽 OCaml プログラミング入門](http://www.geocities.jp/m_hiroi/func/ocaml.html) - 広井誠
 
 
-### openFrameworks
```

Comment:
is there a directory entry that needs removing?

Reply:
There was only one link under openFrameworks which no longer works. Hence I removed it since it was empty.